test/parallel/test-fs-utimes-y2K38.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+
+const assert = require('assert');
+const fs = require('fs');
+
+tmpdir.refresh();
+
+// Check for Y2K38 support. For Windows and AIX, assume it's there. Windows
+// doesn't have `touch` and `date -r` which are used in the check for support.
+// AIX lacks `date -r`.
+if (!common.isWindows && !common.isAIX) {
+  const testFilePath = `${tmpdir.path}/y2k38-test`;
+  const testFileDate = '203801020304';
```

Comment:
This date falls before the y2k38 rollover date (2038-01-19T03:14:08.000Z).

Reply:
So, to be safe, we should pick a date far enough in the future that it doesn't cause timezone issues.